plugins/inputs/win_services/win_services_test.go

```diff
@@ -0,0 +1,101 @@
+// +build windows
+
+//this test must be run under administrator account
+package win_services
+
+import (
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+    "golang.org/x/sys/windows/svc/mgr"
+)
+
+var InvalidServices = []string{"XYZ1@", "ZYZ@", "SDF_@#"}
+var KnownServices = []string{"LanmanServer", "TermService"}
+
+func TestList(t *testing.T) {
+	services, err := listServices(KnownServices)
```

Comment:
That's exactly what I meant. 

You have to create on stuct (proxy) that satifies interface and redirect call to real implementation, 
a lot boilerplate code .  

But if it has to be that way...